modules/sd-install-logging-operator.adoc

```diff
@@ -1,14 +1,26 @@
 
 // Module included in the following assemblies:
 //
-// logging/rosa-install-logging.adoc
+// logging/sd-install-cloudwatch-logging.adoc
```

Comment:
Same thing as sd-install-cloudwatch-logging.adoc, I don't think that assembly is in the preview, so I can't verify the rendering of this file. But I'll do my best with just the source for now.